packages/core/database/lib/metadata/relations.js

```diff
@@ -398,12 +401,20 @@ const createJoinTable = (metadata, { attributeName, attribute, meta }) => {
   const joinColumnName = _.snakeCase(`${meta.singularName}_id`);
   let inverseJoinColumnName = _.snakeCase(`${targetMeta.singularName}_id`);
 
-  // if relation is slef referencing
+  // if relation is self referencing
   if (joinColumnName === inverseJoinColumnName) {
     inverseJoinColumnName = `inv_${inverseJoinColumnName}`;
   }
 
-  metadata.add({
+  const orderColumnName = _.snakeCase(`${meta.singularName}_order`);
```

Comment:
Should this be the opposite naming? if Articles have categories shouldn't this be categories_order here, and the inverse be articles_categories?

Reply:
I think both are possible as we can also see it as : what is the order from the article side?
I think I prefer to inverse however, it's move intuitive